opts/opts_test.go

```diff
@@ -179,6 +180,18 @@ func TestValidateExtraHosts(t *testing.T) {
 	}
 }
 
+func TestIpOptString(t *testing.T) {
+	addresses := []string{"", "0.0.0.0"}
```

Comment:
doi - sorry, I misread the test. For some reason I thought that was just one test where 0.0.0.0 ended up mapping to "".
One day I'll learn to read  :-)